util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -32,32 +32,6 @@
 public class HadoopCredentialConfiguration
     extends EntriesCredentialConfiguration implements Configurable {
 
-  /**
-   * An adapter to use our Configuration object as the config object
-   * for our superclass.
-   */
-  public static class ConfigurationEntriesAdapter
```

Comment:
Let's leave this class here for backward compatibility (even if this class will be empty) and make it extend Hadoop version specific classes.